src/entities/team.ts

```diff
@@ -95,6 +88,7 @@ export class Team extends BaseEntity {
     throw new Error('Unable to retrieve a team due to concurrency issues');
   }
 
+  /* istanbul ignore next */
   static async updateSelectedTeam(team: Team, hash: string): Promise<UpdateResult> {
```

Comment:
@johnkahn I realized that 99% of these methods are just sql... I guess we could check for the values the use, but I'm not sure I see much value in that. Thoughts?